### DISCUSSION TITLE
RSS Button

### DISCUSSION THREAD

**Discussion by huntsyea** [imfing/hextra#370] [General]
How do we get the RSS button similar to the example on our blog? I am not seeing it.

**Comment by imfing** [comment]
https://github.com/imfing/hextra/blob/4a7ab16695502197c5f539690106b24a47532647/exampleSite/content/blog/_index.md?plain=1#L11

**Comment by ahgraber** [comment]
I think I'm having a similar issue.

I've copied this shortcode into /blog/_index.md, but it is still not rendering.  When I look at the rendered index.html, a bunch of `raw html omitted` comments appear where I think the hero-badge is supposed to appear.

I _do_ have `goldmark.renderer.unsafe = true` :)

**Reply by imfing** [reply]
@ahgraber Can you post a screenshot? Not sure if that's caused by hugo omitted the HTML code or the CSS styles not applied.

**Reply by ahgraber** [reply]
<img width="1367" alt="Screenshot 2024-05-05 at 9 37 10 PM" src="https://github.com/imfing/hextra/assets/24922003/ed4ad7a4-4b87-4091-ac81-abbd9a45e677">

blog source is here if it helps: https://github.com/ahgraber/AIMLbling-about

**Reply by imfing** [reply]
@ahgraber the `renderer` key needs to be indented under `goldmark` :)

https://github.com/ahgraber/AIMLbling-about/blob/2534d667db182834f37097a45f22ea91bd643ddd/config/_default/markup.yaml#L23

**Reply by ahgraber** [reply]
D'oh 🤦‍♂️

Thank you. Shows up now!
